RestaurantManagementSystem/RestaurantManagementSystem/src/UseCases/Register/RegisterRequestCommandHandler.cs

```diff
@@ -10,24 +10,21 @@ namespace RestaurantManagementSystem.src.UseCases.Register
 {
     public class RegisterRequestCommandHandler : IRegisterRequestCommandHandler
     {
-        private readonly IRequestRestaurant _requestRestaurant;
+        private readonly IRegistrationRequestRepo registrationRequestRepo;
         private readonly IEmailService _emailService;
 
-        public RegisterRequestCommandHandler(IRequestRestaurant requestRestaurant, IEmailService emailService)
+        public RegisterRequestCommandHandler(IRegistrationRequestRepo _registrationRequestRepo, IEmailService emailService)
         {
-            _requestRestaurant = requestRestaurant;
+            registrationRequestRepo = _registrationRequestRepo;
             _emailService = emailService;
         }
         public void Execute(string name, string email
-            , string address, ContactInfo contact)
+            , string address, string contactinfoemail, string contactinfophonenumber)
```

Comment:
why you are not using the command here instead of receiving the params like this?